src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -113,48 +108,139 @@ To chat with New Relic AI about log errors:
+```
+Summarize this dashboard for me. 
+```
+
+<Callout variant="tip">
+To use NRAI, make sure you're on a usage-based pricing model. NRAI is metered in [Compute Capacity Units](/docs/licenses/license-information/product-definitions/new-relic-one-pricing-definitions#compute-capacity-units). If you aren't on a usage-based pricing model, you can use NRAI with a free 30-day trial. To do this, click **Ask AI** in the top right-hand corner. You will be prompted to review the AI Specific Terms. You can either dismiss or click **Start Trial**. 
+</Callout>
 
-## Secure data
+## Secure data [#secure-data]
 
-Data security and privacy are primary concerns for anyone building and maintaining software. We want you to know that anything you ask our AI is safe. NRAI is built with the same security controls and standards that safeguard New Relic customer data. We will never share your data with other accounts or customers.  We work with vendors that meet our privacy and security standards, and we carefully assess prospective vendors in these critical areas before engaging their services. 
+Our AI uses a variety of inputs to improve its responses and tries to provide the best suggestion to your prompt. For example, it uses inputs like our product documentation, an extensive library of example queries, and your telemetry data. 
 
-Our AI uses a variety of inputs to improve its responses and always provides the best answer. For example, it uses inputs like our product documentation, a large library of example queries, and your telemetry data. But even though our AI scans your data as needed to provide context and help you build charts and understand response times, that data is _never stored_ by New Relic.
+For more information about New Relic's security controls, please see our [Security Policy](docs/licenses/license-information/referenced-policies/security-policy/)
 
-## Always learning 
+## Always learning [#always-learning]
 
-New Relic AI is becoming more accurate and responsive the more it's used. We want to build the AI experience you want. When chatting with our AI, please give its response a thumbs up or a thumbs down.  If you notice something that isn't right or that could be improved, let us know. Go to the user menu, click <DoNotTranslate>**Help**</DoNotTranslate>, then click <DoNotTranslate>**Give us feedback**</DoNotTranslate>.
+New Relic AI is becoming more accurate and responsive the more it's used. We want to build the AI experience you want. When chatting with our AI, please give its response a thumbs up or a thumbs down. If you notice something that needs to be corrected or that could be improved, let us know. Go to the user menu, click <DoNotTranslate>**Help**</DoNotTranslate>, then click <DoNotTranslate>**Give us feedback**</DoNotTranslate>.
 
 Also, New Relic AI can recognize multiple languages and respond in them. If you have use cases for multi-lingual support, please test them with our AI and give us feedback.
 
+<Callout variant="tip">
```

Comment:
@homelessbirds I don't think this needs to be a tip. Instead, consider structuring this content with H3s so they can skim quicker to decide what they want to read